app/views/hyrax/content_blocks/_modal_content_block.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="modal fade" id="change-tab-content-block" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="delete-collection-form">
+        <div class="modal-body">
+          <p>Are you sure you want to leave this tab?  Any unsaved data will be lost.</p>
```

Comment:
Please pull this string out to a local file for internationalization.

Reply:
@cjcolvar - This string has been pull out.  Thanks.